public/app/plugins/datasource/loki/APP_PLUGIN_DOCUMENTATION.md

```diff
@@ -0,0 +1,148 @@
+# Loki data source documentation for app plugins developers
+
+Welcome to the developer documentation for the Loki data source! This documentation is designed to assist app plugin developers in leveraging the power of Loki data source within Grafana. Loki data source provides a set of APIs to simplify common tasks, making it easier to create engaging app plugins.
+
+## Introduction
+
+Loki data source provides a variety of methods, but not all of them are suitable for external use. In this documentation, we will focus on the key APIs that are highly recommended for app plugin development.
+
+It's important to note that the other APIs and methods that are not mentioned here may undergo changes or may not be suitable for external integration. Therefore, we do not recommend relying on them for your development needs.
+
+## Recommended APIs
+
+e strongly advise using these recommended APIs instead of direct proxy calls because they offer several benefits:
```

Comment:
```suggestion
We strongly advise using these recommended methods instead of direct API calls because they offer several benefits:
```

I think it would be positive to make a disticintion between server API and data source methods, so it's easier to know when you're speaking about an XHR request or a method call, to reduce ambiguity.

Even though that, I think it's also completely fine to refer to the set of public methods of the data source as "Loki data source API" or "data source API".